Gemfile

```diff
@@ -167,3 +167,5 @@ gem "image_processing", "~> 1.12"
 gem "prefixed_ids"
 
 gem "mapkick-rb", "~> 0.1.4"
+
+gem "turbo_power", "~> 0.5.0"
```

Comment:
This needs to be in the `.gemspec` file as a runtime dependency

Reply:
Already added on this commit https://github.com/avo-hq/avo/pull/1994/commits/9a09b677a2b53cce3ff6f89f61becd8354c1dec6